### ISSUE TITLE
filter doesn't work without prefix

### ISSUE THREAD

**Issue by codingWithJazz** [LeaVerou/prefixfree#149]

**Comment by nicolasbadia** [comment]
+1

**Comment by LeaVerou** [comment]
Yes, this is a known issue. The problem is that feature detection fails, because browsers already support an unprefixed `filter` property due to SVG. Even if we special case it, we still need to be able to distinguish SVG uses from CSS uses and only prefix the latter. It seems that it would require tons of code, and not with great performance, so maybe it would be better as a plugin. But I'm open to ideas if anyone has any good ones about how to do it...

**Comment by fr0** [comment]
+1. The CSS filter property is the one place in my app where I have to use vendor prefixes, and it sticks out like a sore thumb.

**Comment by ghost** [comment]
Is suppose this is the reason it fails for `clip-path` as well.

**Comment by LeaVerou** [comment]
Yup. :(
